app_dart/lib/src/request_handlers/migrate_checklist.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:googleapis/bigquery/v2.dart';
+import 'package:meta/meta.dart';
+
+import '../datastore/cocoon_config.dart';
+import '../model/appengine/commit.dart';
+import '../request_handling/api_request_handler.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+import '../service/datastore.dart';
+
+@immutable
+class MigrateChecklist extends ApiRequestHandler<MigrateChecklistResponse> {
+  const MigrateChecklist(
+    Config config,
+    AuthenticationProvider authenticationProvider, {
+    @visibleForTesting
+        this.datastoreProvider = DatastoreService.defaultProvider,
+    TabledataResourceApi tabledataResourceApi,
+  })  : assert(datastoreProvider != null),
+        tabledataResourceApi = tabledataResourceApi ?? TabledataResourceApi,
+        super(config: config, authenticationProvider: authenticationProvider);
+
+  final DatastoreServiceProvider datastoreProvider;
+  final TabledataResourceApi tabledataResourceApi;
+
+  @override
+  Future<MigrateChecklistResponse> get() async {
+    const int maxRecords = 10;
```

Comment:
Why 10?

Reply:
Based on our daily commits number, 10 seems to be a reasonably enough number (or use a smaller). I am thinking about running cronjob hourly. What do you think?

Reply:
We should add a comment explaining the rationale for this number. At some point it may need adjustment and it will be helpful to understand what the right count should be.

10 also just seems small to me. Why not 100 or 1000?

Reply:
Based on discussion with @dnfield , increased the number to 100.

Another thing to clarify, this API only copies data from Datastore to Bigquery, rather than moves data. 

The main purpose is for data analysis based on BigQuery.